env-config/config.py

```diff
@@ -44,7 +44,11 @@
     'loggers': {
         'security_monkey': {
             'handlers': ['file', 'console'],
-            'level': 'DEBUG'
+            'level': 'INFO'
```

Comment:
Please revert the changes to this file.

Reply:
Also, can you add some docs on how to set up and configure this?

Reply:
All done, build in progress